Update list of yum packages needed to install, separate pip3 packages (#55)
IBM/ibmi-oss-examples: machinelearning/README.md

# ML on IBM i
## Intro
  This project is used to demonstrate the capacity of IBM i platform on ML/AI area. I encounge you to use, share or even add more sub-projects/notebook to this repo.  Any comments , suggestions or advisors are welcome. thanks.

## How to setup your environment of ML on IBM i
+ Here's where we can  [setup RPM](http://ibm.biz/ibmi-rpms
) env on i. 

> + After setup the RPM env on i, we need to install following packages by yum: `yum install <package name>`. A [detailed instruction](https://www.ibmsystemsmag.com/IT-Strategy/11/2019/How-to-Start-ml-on-IBM-i?utm_content=105130364&utm_medium=social&utm_source=twitter&hss_channel=tw-488711278

Reviewer: ```suggestion
+ After setup the RPM env on i, the following packages need to be installed using yum: `yum install <package name>`. A [detailed instruction](https://www.ibmsystemsmag.com/IT-Strategy/11/2019/How-to-Start-ml-on-IBM-i
```

Author: ok

python3|Python3 support|
python3-devel|python3 development package|
python3-ibm_db|DBAPI support package for IBM i|
python3-numpy|numpy package|
python3-scipy|scipy package|
python3-scikit-learn|scikit-learn package|

> libzmq-devel|Zero-MQ library|

Reviewer: ```suggestion
libzmq-devel|Zero MQ library|
```

Author: ok

freetype-devel|Freetype library|
libjpeg-turbo-devel|JPEG library|
zlib-devel|zlib development package|
libffi-devel|FFI development package|

> + After yum packages are installed, we need to install following Python packages by pip3: `pip3 install <package name>`.

Reviewer: ```suggestion
+ After yum packages are installed, the following Python packages need to be installed using pip3: `pip3 install <package name>`.
```

Author: ok

  `jupyterf notebook --ip=<your host name> --port=<port number>`

## Contents of this repo.

> - **dbconn/loadcsv2db2.py**: this is a tool to import your data from csv to db2 on i. Or course, in most cases your data is already in db2.  NOTE, this script is using the non-journal database.

Reviewer: ```suggestion
- **dbconn/loadcsv2db2.py**: this is a tool to import your data from csv to db2 on i. Of course, in most cases your data is already in db2.  NOTE, this script is using the non-journal database.
```

Author: ok
